src/facets/AccusationLibrary.t.sol

```diff
@@ -3,74 +3,31 @@ pragma solidity >= 0.5.15;
 
 import "ds-test/test.sol";
 
-// import "./AccusationLibrary.sol";
+import "./AccusationLibrary.sol";
 import "./AccusationMultipleProposalFacet.sol";
 
-contract Foo {
-    function bomb(bool b) public {
-        require(b == false);
-    }
-}
-
 contract TestAccusationLibrary is DSTest {
 
-
-
-    function testFoo() public {
-        Foo b = new Foo();
-
-        bool ok;
-
-        // this invocation of bomb() is successful
-        (ok, ) = address(b).delegatecall(abi.encodeWithSignature("bomb(bool)", false));
-        assertTrue(ok, "function successful");
-
-        // this invocation of bomb() reverts
-        (ok, ) = address(b).delegatecall(abi.encodeWithSignature("bomb(bool)", true));
-
-        assertTrue(!ok, "function reverted");
-    }
-
-    function testAccuseMultipleProposal() public {
+    function testSignNoPrefix() public {
         AccusationMultipleProposalFacet f = new AccusationMultipleProposalFacet();
 
-        // bytes memory sig0 = hex"e9be13c2658fcc1769b5e85e6e238a906fbf0d37c52a4ad5f4c69ff18156f3cb093a78ec7e848ab0649cfd1de13cfa4f2b215e42596f9c415541ec813b4626b401";
-        bytes memory sig0 = hex"cba766e2ba024aad86db556635cec9f104e76644b235f77759ff80bfefc990c5774d2d5ff3069a5099e4f9fadc9b08ab20472e2ef432fba94498d93c10cc584b00";
+        bytes memory sig = hex"cba766e2ba024aad86db556635cec9f104e76644b235f77759ff80bfefc990c5774d2d5ff3069a5099e4f9fadc9b08ab20472e2ef432fba94498d93c10cc584b00";
+        bytes memory prefix = "";
+        bytes memory message = hex"54686520717569636b2062726f776e20666f782064696420736f6d657468696e67";
```

Comment:
Is this a hash of a Pclaims that had "Proposal" appended to it?

Reply:
No, this was just the string "The quick brown fox did something" with no prefix. The test `testSignedPClaims()` is a real pclaims with "Proposal" prepended.